community/modules/scheduler/schedmd-slurm-gcp-v6-controller/main.tf

```diff
@@ -40,6 +40,31 @@ locals {
 
 }
 
+locals {
+  synt_suffix       = substr(md5("${var.project_id}${var.deployment_name}"), 0, 5)
+  synth_bucket_name = "${local.slurm_cluster_name}${local.synt_suffix}"
+}
+
+module "bucket" {
+  source  = "terraform-google-modules/cloud-storage/google"
+  version = "~> 3.0"
+
+  count = var.create_bucket ? 1 : 0
+
+  location   = var.region
+  names      = [local.synth_bucket_name]
+  prefix     = "slurm"
+  project_id = var.project_id
+
+  force_destroy = {
+    (local.synth_bucket_name) = true
+  }
+
+  labels = {
+    slurm_cluster_name = local.slurm_cluster_name
```

Comment:
nit: Ideally we would add resource tracking labels here. I notice that controller does not take labels. I think it should.

I am fine to add in separate PR, but please follow up.

Reply:
tracking as a separate issue.